Dockerfile

```diff
@@ -26,16 +32,21 @@ ADD . /app/fonzie/
 
 # Install application and project requirements
 RUN cd /app/fonzie && \
-    pip install --exists-action w -r requirements.txt
+    pip install --exists-action w -r requirements-dev.txt
 
 # FIXME: pyopenssl seems to be linked with a wrong openssl release leading to
 # bad handskake ssl errors. This looks ugly, but forcing pyopenssl
 # re-installation solves this issue.
 RUN pip install -U pyopenssl
 
+# Create pytest cache directory and give Docker user write permissions
+RUN mkdir -p /edx/app/edxapp/edx-platform/.pytest_cache && \
+    chown $user:$group /edx/app/edxapp/edx-platform/.pytest_cache
```

Comment:
This should not be required as you mount it in docker-compose

Reply:
True. My bad, sorry.